.github/workflows/code_checks.yml

```diff
@@ -21,12 +21,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      # Sets up node with v16 and restores global cache so we don't have to re-download dependencies
       - uses: actions/setup-node@v3
         with:
-          node-version: '16'
-          # cache: 'yarn'
-          # cache-dependency-path: 'VAMobile/yarn.lock'
```

Comment:
Not sure why this was originally commented out, but is there a reason why we're not using the yarn cache in CI?

Reply:
I believe it was causing some weirdness with the yarn install, but not 100% sure. I didn't want to mess with it for this PR